mono/mini/debugger-agent.c

```diff
@@ -3690,6 +3817,9 @@ process_event (EventKind event, gpointer arg, gint32 il_offset, MonoContext *ctx
 		case EVENT_KIND_STEP:
 			buffer_add_methodid (&buf, domain, (MonoMethod *)arg);
 			buffer_add_long (&buf, il_offset);
+#if defined(IL2CPP_MONO_DEBUGGER) && defined(IL2CPP_DEBUGGER_TESTS)
+			buffer_add_long (&buf, il2cpp_seqpoint_id);
```

Comment:
Why are these only needed by the tests?

Reply:
This is a change in the protocol that we would need to push to all of the clients. We don't want to do that. It is useful to see the sequence point IDs in the debugger of the test code while were are developing though. That helps us build a mental map of the location the test code thinks we should be to the actual location where the debugger stopped execution.